Gems/AWSClientAuth/Code/Include/Private/Authorization/AWSCognitoAuthorizationController.h

```diff
@@ -9,6 +9,7 @@
 #pragma once
 
 #include <Authorization/AWSCognitoAuthorizationBus.h>
+#include <Authorization/AWSClientAuthCognitoCachingAuthenticatedCredentialsProvider.h> // new
```

Comment:
Nit: Please remove the comment when you merge this PR :)

Reply:
d'oh! thanks for catching, will do.